src/main/webapp/archives.html

```diff
@@ -558,12 +558,20 @@ <h3 class="text-dark">Karaf Cellar</h3>
                     </thead>
                     <tbody>
                         <tr>
-                        <td>4.1.0</td>
-                        <td><a href="javascript:var w = window.open('download.html')">Installation Instructions</a></td>
-                        <td><a href="http://archive.apache.org/dist/karaf/cellar/4.1.0/apache-karaf-cellar-4.1.0-src.tar.gz">tar.gz</a> [<a href="http://archive.apache.org/dist/karaf/cellar/4.1.0/apache-karaf-cellar-4.1.0-src.tar.gz.asc">PGP</a>] [<a href="http://archive.apache.org/dist/karaf/cellar/4.1.0/apache-karaf-cellar-4.1.0-src.tar.gz.md5">MD5</a>] [<a href="http://archive.apache.org/dist/karaf/cellar/4.1.0/apache-karaf-cellar-4.1.0-src.tar.gz.sha1">SHA1</a>]<br>
-                                    <a href="http://archive.apache.org/dist/karaf/cellar/4.1.0/apache-karaf-cellar-4.1.0-src.zip">zip</a> [<a href="http://archive.apache.org/dist/karaf/cellar/4.1.0/apache-karaf-cellar-4.1.0-src.zip.asc">PGP</a>] [<a href="http://archive.apache.org/dist/karaf/cellar/4.1.0/apache-karaf-cellar-4.1.0-src.zip.md5">MD5</a>] [<a href="http://archive.apache.org/dist/karaf/cellar/4.1.0/apache-karaf-cellar-4.1.0-src.zip.sha1">SHA1</a>]
-                        </td>
-                        <td><a href="javascript:var w = window.open('https://issues.apache.org/jira/secure/ReleaseNote.jspa?projectId=12311140&version=12340615')">Release Notes</a></td>
+                            <td>4.1.1</td>
+                            <td><a href="javascript:var w = window.open('download.html')">Installation Instructions</a></td>
+                            <td><a href="http://archive.apache.org/dist/karaf/cellar/4.1.1/apache-karaf-cellar-4.1.1-src.tar.gz">tar.gz</a> [<a href="http://archive.apache.org/dist/karaf/cellar/4.1.1/apache-karaf-cellar-4.1.1-src.tar.gz.asc">PGP</a>] [<a href="http://archive.apache.org/dist/karaf/cellar/4.1.1/apache-karaf-cellar-4.1.1-src.tar.gz.md5">MD5</a>] [<a href="http://archive.apache.org/dist/karaf/cellar/4.1.1/apache-karaf-cellar-4.1.1-src.tar.gz.sha1">SHA1</a>]<br>
```

Comment:
That's not correct anymore. Per "new" Apache rule, `md5` is not used anymore and `sha1` is replaced by `sha512`.

Reply:
Ok, I made the changes on the Cellar 4.1.1 archives.